src/components/tools/details/Drawer.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * @author sriram
+ *
+ * A drawer that display tabs with various information such as job history, params etc...
+ *
+ */
+
+import React, { useState } from "react";
+
+import { useQuery } from "react-query";
+import { useTranslation } from "i18n";
+
+import { build } from "@cyverse-de/ui-lib";
+
+import ids from "../ids";
+import constants from "../../../constants";
+import DETabPanel from "../../utils/DETabPanel";
+import ToolDetails from "./ToolDetails";
+import TableView from "components/apps/listing/TableView";
+
+import {
+    getToolDetails,
+    getAppsUsed,
+    TOOL_DETAILS_QUERY_KEY,
+    APPS_USING_QUERY_KEY,
+} from "serviceFacades/tools";
+
+import { Drawer, Tab, Tabs, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+const TABS = {
+    toolDetails: "TOOL DETAILS",
+    appsUsingTool: "APPS USING TOOL",
+};
+
+const useStyles = makeStyles((theme) => ({
+    drawerPaper: {
+        [theme.breakpoints.up("lg")]: {
+            maxWidth: "25%",
+        },
+        [theme.breakpoints.down("lg")]: {
+            maxWidth: "50%",
+        },
+        [theme.breakpoints.down("sm")]: {
+            maxWidth: "90%",
+        },
+    },
+
+    drawerHeader: {
+        margin: theme.spacing(1),
+        display: "flex",
+        flexDirection: "row",
+        maxWidth: "100%",
+    },
+    drawerSubHeader: {
+        marginLeft: theme.spacing(2),
+        display: "flex",
+        flexDirection: "row",
+        maxWidth: "100%",
+    },
+
+    tabIndicator: {
+        backgroundColor: theme.palette.secondary.main,
+        color: theme.palette.secondary.contrastText,
+    },
+    tabSelected: {
+        color: theme.palette.primary.contrastText,
+        backgroundColor: theme.palette.primary.main,
+    },
+    headerOperations: {
+        marginLeft: theme.spacing(1),
+    },
+}));
+
+function DetailsDrawer(props) {
+    const classes = useStyles();
+    const { t } = useTranslation("tools");
+    const { selectedTool, open, onClose, baseId } = props;
+    const [selectedTab, setSelectedTab] = useState(TABS.toolDetails);
+    const [details, setDetails] = useState();
+    const [apps, setApps] = useState();
+
+    const drawerId = build(baseId, ids.DETAILS_DRAWER);
+    const infoTabId = build(drawerId, ids.INFO_TAB);
+    const paramsTabId = build(drawerId, ids.APPS_USING_TOOL);
```

Comment:
It might make more sense to name this `appsTabId`, or something like that.

Reply:
Oops..Copy Pasta issue.